tidy3d/version.py

```diff
@@ -1,3 +1,3 @@
 """DO NOT EDIT: Modified automatically with .bump2version.cfg"""
 
-__version__ = "2.8.0rc1"
+__version__ = "2.9.0rc2"
```

Comment:
What was this change about? Just so you can map to your solver? :)

That said, we may seriously consider postponing this for 2.9 if we don't settle on a good scalable way to implement this that works for RF + optical simulations.

Reply:
Yeah, this is for RF folks to run simulations with SIBC. The benchmark is undergoing now.

Reply:
I'd still aim for 2.8; otherwise there can be lots of conflicts to resolve